src/main/webapp/app/app.tsx

```diff
@@ -63,6 +62,7 @@ const mapStoreToProps = ({ authStore, layoutStore, firebaseAppStore }: IRootStor
   authorities: authStore.account.authorities,
   isCurator: hasAnyAuthority(authStore.account.authorities ?? [], [AUTHORITIES.CURATOR]),
   loadingAuth: authStore.loading,
+  authAccount: authStore.account,
```

Comment:
Use `authStore.isAuthenticated` instead?

Reply:
@calvinlu3 I think it should be ok, but based on the logic in line 48, we still want to render the routes even the user is not authorized?

Reply:
Yeah there are some routes that should be available when unauthorized like login and logout. The other routes are wrapped as PrivateRoutes, so they will check for authorities.

Reply:
If that's the case, I can't use isAuthenticated then. I will not render the login/logout.